src/server/schema/resolvers/queries/getLife.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers } from '../definitions';
+
+const query: GraphQLQueryResolvers['getLife'] = async (root, { id }) => {
+    const { collections } = await getDatabaseContext();
+    const lifeID = new ObjectId(id);
+    const lifeById = await collections.lives.findOne({'_id': lifeID});
```

Comment:
you can instead directly return this. This will return undefined if it does not exist which is still ok since life is optional. `collections.lives.findOne({_id: lifeID});`

Reply:
Alright! have changed. Essentially this is because the action of returning undefined is not an error and thus not necessary?